app/ui/downloadCompleted.js

```diff
@@ -10,15 +10,7 @@ module.exports = function(state) {
       <h1 class="text-center text-3xl font-bold my-2">
         ${state.translate('downloadFinish')}
       </h1>
-      <img src="${assets.get('completed.svg')}" class="my-12 h-48" />
-      <p class="text-grey-80 leading-normal dark:text-grey-40">
-        ${state.translate('trySendDescription')}
-      </p>
-      <p class="my-5">
-        <a href="/" class="btn rounded-lg flex items-center mt-4" role="button"
-          >${state.translate('sendYourFilesLink')}</a
-        >
-      </p>
+      <img src="${assets.get('checkr-completed.png')}" class="my-12 h-48" />
```

Comment:
hm, I think we would've liked to use the same filenames where we can to make it easier to sync with upstream.  But since they're .svg that's a little bit tricky.  This works.